bigquery_etl/run_multipart_query.py

```diff
@@ -157,6 +162,17 @@ def main():
             )
         )
         try:
+            schema_update_options = []
+
+            if args.schema_update_option == "ALLOW_FIELD_ADDITION":
+                schema_update_options = [
+                    bigquery.SchemaUpdateOption.ALLOW_FIELD_ADDITION
+                ]
+            elif args.schema_update_option == "ALLOW_FIELD_RELAXATION":
+                schema_update_options = [
+                    bigquery.SchemaUpdateOption.ALLOW_FIELD_RELAXATION
+                ]
+
```

Comment:
It looks like there are only [2 schema update options](https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.SchemaUpdateOption.html) that we need to consider here